configure

```diff
@@ -217,12 +217,25 @@ install_local_rebar() {
     fi
 }
 
+install_local_emilio() {
+    if [ ! -x "${rootdir}/bin/emilio" ]; then
+        if [ ! -d "${rootdir}/src/emilio" ]; then
```

Comment:
Probably need to add `src/emilio` and `bin/emilio` to `.gitgnore` with this approach.